kedro_mlflow/framework/hooks/pipeline_hook.py

```diff
@@ -177,6 +185,9 @@ def on_pipeline_error(
             mlflow.end_run()
 
 
+mlflow_pipeline_hooks = MlflowPipelineHook()
```

Comment:
Can you remove this line too? It is also linked to the auto-registration PR.

Reply:
Same for node_hook